src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -1698,6 +1731,9 @@ def __call__(
         if head_mask is None:
             head_mask = jnp.ones((self.config.num_hidden_layers, self.config.num_attention_heads))
 
+        if indices_prng_key is None:
+            indices_prng_key = jax.random.PRNGKey(0)
+
         # Handle any PRNG if needed
         rngs = {}
```

Comment:
This is how we should pass the indices PRNG key. Not as an argument, but rather in the dict of RNGs. This is the same logic that we use for dropout (see below):

```suggestion
        rngs = {}
        if indices_prng_key is not None:
            rngs["indices"] = indices_prng_key
```